src/seedu/addressbook/commands/fees/EditFeesCommand.java

```diff
@@ -8,13 +11,14 @@
 import seedu.addressbook.data.person.Person;
 import seedu.addressbook.data.person.ReadOnlyPerson;
 import seedu.addressbook.data.person.UniquePersonList.PersonNotFoundException;
+import seedu.addressbook.data.tag.Tag;
 
 /**
  * Adds fees to a respective person
  */
-public class AddFeesCommand extends Command {
+public class EditFeesCommand extends Command {
 
-    public static final String COMMAND_WORD = "addfees";
+    public static final String COMMAND_WORD = "editfees";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n" + "Adds fees to an existing person "
```

Comment:
I think you missed to update the message usage here. From add fees to edit fees.

Reply:
This too!!